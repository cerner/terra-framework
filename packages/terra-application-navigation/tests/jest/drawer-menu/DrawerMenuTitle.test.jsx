import React from 'react';
import DrawerMenuTitle from '../../../src/drawer-menu/_DrawerMenuTitle';

describe('DrawerMenuTitle', () => {
  it('should render default element', () => {
    const shallowComponent = shallow(
      <DrawerMenuTitle
        titleConfig={{
          title: 'test-title',
        }}
      />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });
});
