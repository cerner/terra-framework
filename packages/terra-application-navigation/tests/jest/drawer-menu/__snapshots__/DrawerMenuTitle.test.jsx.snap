// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerMenuTitle should render default element 1`] = `
<div
  className="title-container"
>
  <div
    className="title"
    title="test-title"
  >
    test-title
  </div>
</div>
`;
